lambda/recipes-responder/src/main.ts

```diff
@@ -48,11 +49,12 @@ export async function processRecord(r:KinesisStreamRecord):Promise<number> {
   }
 }
 
-export const handler:KinesisStreamHandler = async (event) => {
-  const updatesPerEvent = await Promise.all(event.Records.map((r)=>processRecord(r)));
-  const updatesTotal = updatesPerEvent.reduce((acc, current)=>acc+current, 0);
-  if(updatesTotal>0) {
+export const handler: KinesisStreamHandler = async (event) => {
+  const updatesPerEvent = await Promise.all(event.Records.map((r) => processRecord(r)));
+  const updatesTotal = updatesPerEvent.reduce((acc, current) => acc + current, 0);
+  if (updatesTotal > 0) {
     console.log(`Processed updates for ${updatesTotal} recipes, rebuilding the index json`);
+    await registerMetric("UpdatesTotalOfRecipes", updatesTotal)
```

Comment:
I believe that there is already a test for `main.ts` - please assert that `registerMetric()` is correctly called in there

Reply:
`main.ts` for testing handler is implemented to test `registerMetric` call.

Since direct test for handler (where registerMetric is present) was not available in main.ts hence need to be reviewed whther this implementation is right way to test with right mocking (event, context and callback) for handler method to test KinesisStreamEvent as input. thanks.